doc/release-notes-33106.md

```diff
@@ -0,0 +1,17 @@
+Mining and Transaction Relay Policy
+=========================
+
+The minimum block feerate (`-blockmintxfee`) has been changed to 1 satoshi per kvB. It can still be changed using the
+configuration option.
+
+The default minimum relay feerate (`-minrelaytxfee`) and incremental relay feerate (`-incrementalrelayfee`) have been
+changed to 100 satoshis per kvB. They can still be changed using their respective configuration options, but it is
+recommended to change both together if you decide to do so.
+
+Other minimum feerates (e.g. the dust feerate, the minimum returned by the fee estimator, and all feerates used by the
+wallet) remain unchanged. The mempool minimum feerate still changes in response to high volume but more gradually, as a
+result of the change to the incremental relay feerate.
```

Comment:
I wouldn't say this is meaningfully more gradual -- if your mempool fills up with 300MB of 41sat/vb txs, you'll bump your min fee from 0.1 sat/vb to 41.1 sat/vb instead of from 1 sat/vb to 42 sat/vb; and when decreasing it, you'll still decrease it according to a 12 hour halflife (or 6 or 3 hours if your mempool has emptied substantially in the meantime).

Reply:
Removed this part in #33189